lib/src/logic.dart

```diff
@@ -575,6 +575,89 @@ class Logic {
   /// Greater-than-or-equal-to.
   Logic operator >=(dynamic other) => GreaterThanOrEqual(this, other).out;
 
+  /// Shorthand Increment.
+  /// By default for a [Logic] var, if no [incrVal] is provided result is ++var
+  /// else result is var+=[incrVal]
+  ///
+  /// ```dart
+  ///
+  /// Logic a = Logic(width: 8);
+  /// Logic b = Logic(width: 8);
+  /// a = addInput('a', a, width: 8);
+  /// b = addInput('b', b, width: 8);
+  /// final piOut = addOutput('piOut', width: 8);
+  ///
+  /// Combinational([
+  ///   piOut < a,
+  ///   piOut.incr(b),
+  /// ]);
+  ///
+  /// ```
+  ///
+  ConditionalAssign incr([Logic? incrVal]) =>
```

Comment:
It would be nice to support `dynamic` here, since we could want to add by an `int`, `LogicValue`, or `BigInt` as well.  I think all of the math operations (`+-/*`) will already support `dynamic` and properly convert as needed.